Chargebee/Classes/Purchase/CBPurchaseManager.swift

```diff
@@ -302,10 +307,11 @@ public extension CBPurchase {
         }
     }
     
-    func validateReceipt(_ product: SKProduct?,completion: ((Result<(status:Bool, subscriptionId:String?, planId:String?), Error>) -> Void)?) {
+    func validateReceipt(_ product: CBProduct?,completion: ((Result<(status:Bool, subscriptionId:String?, planId:String?), Error>) -> Void)?) {
```

Comment:
Also since this is a public API change, lets plan to communicate it accordingly.

Reply:
`validateReceipt`  till now its used internally from now it will be public method so will update documentation and we will can communicate  accordingly thanks

Reply:
Can we create an alias/type for the tuple here?

Reply:
 we can create but now its align similar to other functions in CBPurchaseManager.Swift and maintaining consistency
yea NonSubscription this we have Struct already  but for validateReceipt  (Result<(status:Bool, subscriptionId:String?, planId:String?), Error>) -> Void)? this is similar to other functions so i think we are good with this not required updation